From 75c3e74ea35219f257c80daf9a236e0f518a7875 Mon Sep 17 00:00:00 2001
Patch-Source: https://github.com/dotnet/source-build/discussions/3176
From: Antoine Martin <dev@ayakael.net>
Date: Fri, 16 Dec 2022 20:16:03 +0000
Subject: [PATCH 1/1] Downgrades Microsoft.DotNet.Build.Tasks.TargetFramework.Sdk

This version of Microsoft.DotNet.Build.Tasks.TargetFramework.Sdk is broken
For some reason, it insists on trying to target every framework, which breaks
build of runtime.

---
diff --git a/src/runtime/global.json.orig b/src/runtime/global.json
index 22aecea4b48..cd4eccec952 100644
--- a/src/runtime/global.json.orig
+++ b/src/runtime/global.json
@@ -12,7 +12,7 @@
     "python3": "3.7.1"
   },
   "msbuild-sdks": {
-    "Microsoft.DotNet.Build.Tasks.TargetFramework.Sdk": "6.0.0-beta.22564.9",
+    "Microsoft.DotNet.Build.Tasks.TargetFramework.Sdk": "6.0.0-beta.22512.3",
     "Microsoft.DotNet.Arcade.Sdk": "6.0.0-beta.22564.9",
     "Microsoft.DotNet.Helix.Sdk": "6.0.0-beta.22564.9",
     "Microsoft.DotNet.SharedFramework.Sdk": "6.0.0-beta.22564.9",
